stable/airflow/templates/configmap-scripts.yaml

```diff
@@ -20,3 +20,17 @@ data:
     else
       exit 0
     fi
+{{- if .Values.airflow.connections }}    
+  add-connections.sh: |
+    #!/bin/sh -e
+    {{- range .Values.airflow.connections }}
+    airflow connections --add --conn_type {{ .type }} --conn_id {{ .id }} 
+    {{- if .uri }} --conn_uri {{ .uri | quote }} {{ end -}}
+    {{- if .host }} --conn_extra {{ .host }} {{ end -}}
```

Comment:
you've got wrong argument name 😅`--conn_extra`

Reply:
Good catch! Thanks. Fixing ASAP.